### ISSUE TITLE
Use something other than mouse to fire 

### ISSUE THREAD

**Issue by code-11** [code-11/CyberSamurai#18]
Credit to @jceipek

Things to try:
-Spacebar 
-Simply auto attack when close to enemies (?)<-would make positioning more important but take away exhilaration of attacking

**Comment by jceipek** [comment]
Using the mouse to attack is a great idea if you use the mouse to move.

**Comment by code-11** [comment]
Ah, I see I've mixed paradigms. Either mouse it up all the way, or keyboard it up, but mixing in this context makes less sense.

**Comment by jceipek** [comment]
Yup. That's because the mouse is a 2d pointing device, so it only really makes sense to use it if you're making use of those capabilities. Top-down 2d shooters make sense with the mouse because you can click on what you want to attack anywhere on the screen. In this prototype, you can only affect things directly around your avatar.
